modules/devguide/examples/StartUsingCapella.java

```diff
@@ -8,33 +8,27 @@
 import java.time.Duration;
 // end::imports[]
 
-
-public class StartUsingCloud {
+public class StartUsingCapella {
   // tag::connect-info[]
   // Update these variables to point to your Couchbase Capella instance and credentials.
-  static String connectionString = "cb.<your-endpoint-here>.cloud.couchbase.com";
+  static String address = "cb.<your-endpoint-here>.cloud.couchbase.com";
```

Comment:
After thinking myself in circles on this one, I do think `connectionString` is probably still best here, or refactor a bit more (per my next comments). Technically the full connection string includes the scheme, so I get what you meant here, but maybe we just make this the full connection string instead? I'm thinking from the sense that it's not really a "hostname" or "address" either but rather our "connection string format" as documented throughout (e.g., https://docs.couchbase.com/java-sdk/current/howtos/managing-connections.html#connection-strings). Our other samples are a bit hit and miss in this regard as well, but several just put the whole string in (or don't even have a variable) and use the full "connection string" here.

Reply:
Done!